generators/app/templates/src/services/AuthServices.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiResponse } from 'apisauce';
+
+import { User, Credentials } from '~app/contexts/UserContext/reducer';
+
+import api from '../config/api';
+
+import LocalStorageService from './LocalStorageService';
+
+const TOKEN_FIELD_NAME = 'sessionToken';
+
+export interface LoginError {
+  message: string;
+}
+
+export const setCurrentUser = (currentUser: User) => {
+  api.setHeader('Authorization', currentUser.sessionToken);
+  LocalStorageService.setValue(TOKEN_FIELD_NAME, currentUser.sessionToken);
+};
+
+export const getCurrentUser = () => LocalStorageService.getValue(TOKEN_FIELD_NAME);
+
+export const removeCurrentUser = () => LocalStorageService.removeValue(TOKEN_FIELD_NAME);
```

Comment:
we should add a logout in the home that calls this and calls the api to invalidate the token